### ISSUE TITLE
[mingw-w64-fox] update to 1.7.84

### ISSUE THREAD

**Issue by iahung3** [msys2/MINGW-packages#19718]
### Package name

mingw-w64-fox

### New version number

1.7.84

### Other information that may be useful

Source tarball: http://fox-toolkit.org/ftp/fox-1.7.84.tar.gz

### Are you willing to submit a PR?

No

**Comment by Biswa96** [comment]
version 1.7.84 is masked as development version here http://www.fox-toolkit.org/

**Comment by iahung3** [comment]
> version 1.7.84 is marked as development version here http://www.fox-toolkit.org/

Their naming scheme sucks. The so called development version is what really being used by people now. You could think it's like Debian versioning scheme: FOX stable = Debian stable, FOX development = Debian Testing, and FOX snapshot = Debian Unstable.

FOX stable is no longer maintained. People with old code base that still using it having to patch it themselves. The FOX developer no longer care about it.

Long story short, FOX toolkit is semi-rolling release like Debian Testing.

You can see, people all target FOX development in their customized build system of FOX:

https://github.com/franko/fox

https://github.com/devinsmith/fox

Btw, FOX still uses autotools which sucks, this is the reason for these alternative build systems of FOX to be created. I'm currently use the Meson based build system as the cmake based build system is not as actively maintained.

**Comment by iahung3** [comment]
@Biswa96 I created a package request for FOX 1.7: https://github.com/msys2/MINGW-packages/issues/19725. This way, we can keep FOX stable but also have the latest version of FOX. Please let me know what you think about it.

**Comment by iahung3** [comment]
With franko's Meson based build system, building from source is so easy. Closed.
